ecl/hql/hqlutil.cpp

```diff
@@ -5803,6 +5803,8 @@ static bool splitDatasetAttribute(SharedHqlExpr & dataset, SharedHqlExpr & attri
     SplitDatasetAttributeTransformer transformer;
     return transformer.split(dataset, attribute, queryNonAliased(expr));
 #else
+    //The following code only matches a subset of the possible expressions.  This can mean that some parts of the expression
```

Comment:
not related to the change, but not worth an issue in its own right.

Reply:
@ghalliday Did you raise one?

Reply:
Yes, HPCC-23013